app/src/main/java/com/github/polypoly/app/utils/global/GlobalInstances.kt

```diff
@@ -34,12 +38,14 @@ class GlobalInstances {
             }
         }
 
-        var currentUser : User = User(7, "fake_user", "I am fake until google db is fully setup", Skin.default(),
+        var currentUser : User = User(15, "fake_user", "I am fake until google db is fully setup", Skin.default(),
```

Comment:
Is there a specific reason for this change?

Reply:
just for testing

Reply:
Then I don't think it should be included in the pr if it is for testing...